src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -1113,6 +1095,32 @@ Reader.SingleResult ToResult(TimeSeriesOperation.AppendOperation element)
             return context.LastDatabaseChangeVector;
         }
 
+        private bool ValueTooFar(TimeSeriesSegmentHolder segmentHolder, Reader.SingleResult current)
+        {
+            var deltaInMs = (current.Timestamp.Ticks / 10_000) - segmentHolder.BaselineMilliseconds;
```

Comment:
Can we wrap this in something similar to `Size`
I'm afraid that without that it is a potential place for easy bug introduction. Let's talk about this tomorrow.

Reply:
Seems reasonable to do so, yes

Reply:
Let's do in a separate issue: https://issues.hibernatingrhinos.com/issue/RavenDB-14577